site/src/pages/TemplateVersionEditorPage/MonacoEditor.tsx

```diff
@@ -13,22 +12,6 @@ export const MonacoEditor: FC<{
   onChange?: (value: string) => void;
 }> = ({ onChange, value, path }) => {
   const theme = useTheme();
-  const [editor, setEditor] = useState<editor.IStandaloneCodeEditor>();
-  useLayoutEffect(() => {
```

Comment:
To resize the editor when the bottom panel was open/closed but Idk why we were using it since the autoLayout prop was set to true 🤔 maybe someone faced a bug and used the listener? I tested both cases and they worked without it tho